scripts/ovm/run-optimism.sh

```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
+# download pre-builted images
+repo="ethereumoptimism"
+for image in {builder,l2geth,deployer};do docker pull $repo/$image; done
+
 # build in 2 steps
 function build_images() {
-  docker-compose build --parallel -- builder l2geth l1_chain
-  docker-compose build --parallel -- deployer dtl batch_submitter relayer
```

Comment:
so we dont need to build these ones as they are pulled from `ethereumoptimism`, right?

Reply:
yeah, but we are able to pull just 3 images, and the other 4 images we need build it.

Reply:
I see. nice optimisation